### DISCUSSION TITLE
Is 'cx_freez'  version 6.5 is available?

### DISCUSSION THREAD

**Discussion by Siddu-Bileyali** [marcelotduarte/cx_Freeze#2453] [Q&A]
Hello All,

I cannot install the cx_freez version 6.5 due to the error below; however, it worked earlier. I just now checked the cx_freez version 6.5 at https://app.readthedocs.org/projects/cx-freeze/, and it was unavailable.

`ERROR: Could not find a version that satisfies the requirement cx-Freeze==6.5 (from versions: 4.3.4, 5.0, 5.1.1, 6.0, 6.1, 6.2, 6.3, 6.4.2, 6.5.3, 6.6, 6.7, 6.8.4, 6.9, 6.10, 6.11.1, 6.12.0, 6.13.2)
ERROR: No matching distribution found for cx-Freeze==6.5
`
Is it deleted/renamed ?

Thank you,
Siddu

**Comment by marcelotduarte** [comment]
Unfortunately, I'm having to delete some old versions to be able to publish new ones, due to a quota on PyPI. I've already asked for a quota increase, but they haven't responded.
I recommend you use `cx_Freeze==6.5.3` or `cx_Freeze~=6.5` or try a newer version.

**Reply by Siddu-Bileyali** [reply]
Thanks a lot for quick reply..!
If you don't mind could you please let me know when exactly will it be deleted 

Thanks in Advance!

**Reply by marcelotduarte** [reply]
I didn't keep the emails, but it was for the publication of version 7.0 which occurred on April, 21, or for 7.1 on May, 25.

**Comment by Siddu-Bileyali** [comment]
Thank you..
